src/context.cc

```diff
@@ -2273,6 +2273,29 @@ error Context::Signup(
     return Login(email, password);
 }
 
+error Context::GoogleSignup(
+    const std::string access_token,
+    const uint64_t country_id) {
+
+    TogglClient client(UI());
+    std::string json("");
```

Comment:
Since we don't use JSON in this func. I suggest we could get rid of it.

Reply:
We use it on the next line as the third parameter to `signupGoogle` method.
I'd say let's leave it like that as it's quite readable and we're using it like this in other places.